components/Link/Link.test.tsx

```diff
@@ -21,15 +21,13 @@ describe("Link", () => {
 
   it("HandleClick is run on link click", () => {
     const handleClick = jest.fn();
-    const { getByRole } = render(
-      <Link href={""} appearance={"button"}>
-        test
-      </Link>
-    );
+    const href = "";
+    const { getByRole } = render(<Link appearance={"button"}>test</Link>);
```

Comment:
Aca vamos a pasarle el `href` al link, asi:
```suggestion
    const { getByRole } = render(<Link href="/" appearance={"button"}>test</Link>);
```
Ya no hace falta la constante `const href = "";`

Reply:
check